packages/babel-runtime/helpers/toArray.js

```diff
@@ -1,7 +0,0 @@
-var _Array$from = require("../core-js/array/from");
```

Comment:
I think it's helpful to have helpers that test this core-js injection behavior. Why don't we leave the toArray helper and just add the temporalRef one as well.